### DISCUSSION TITLE
Set an environment variable for the menu light/dark state

### DISCUSSION THREAD

**Discussion by gilbarbara** [swiftbar/SwiftBar#83] [General]
Hey!

I've seen that you can set the `color` for dark/light menus, but it would great to have the same option for images too.
Maybe an environment variable would be better for this case (and possible others).

Thanks~

**Comment by melonamin** [comment]
Adding env variable sounds good to me, added #87 

As for different images, probably yes? Honestly I don't have opinion on that, I mostly use SF Symbols and it's look great in light\dark :).

**Reply by gilbarbara** [reply]
The env variable is perfect, thank you
